Tools/i18n/msgfmt.py

```diff
@@ -158,14 +158,19 @@ def make(filename, outfile):
             msgctxt = b''
         elif l.startswith('msgid') and not l.startswith('msgid_plural'):
             if section == STR:
-                add(msgctxt, msgid, msgstr, fuzzy)
-                msgctxt = None
                 if not msgid:
+                    # Filter out POT-Creation-Date
+                    # See issue #131852
+                    msgstr = ''.join(line for line in msgstr.decode(encoding).splitlines(True)
+                                     if 'POT-Creation-Date' not in line).encode(encoding)
```

Comment:
You can operate directly on the bytestring without decoding/encoding (`msgstr` should also be a bytestring rather than a string):
```suggestion
                    msgstr = b''.join(line for line in msgstr.splitlines(True)
                                      if not line.startswith(b'POT-Creation-Date'))
```